XPlat/RadialProgress/nuget/Xamarin.RadialProgress.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.RadialProgress</id>
+    <title>Xamarin Radial Progress</title>
+    <version>1.0.2</version>
+    <authors>Xamarin Inc.</authors>
+    <owners>Xamarin Inc.</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>A stylish, circular progress view for your Xamarin iOS and Android applications</description>
+    <copyright>Copyright 2016 Xamarin</copyright>
```

Comment:
I have asked, and I have not been given good answers.  It is my understanding that it is not wrong.  Leaving as Xamarin.